types/vochain.go

```diff
@@ -128,3 +128,20 @@ type ScrutinizerOnProcessData struct {
 	EntityID  []byte
 	ProcessID []byte
 }
+
+// _________________________ CENSUS ORIGINS __________________________
+
+type CensusProperties struct {
+	Name            string
+	NeedsDownload   bool
+	NeedsIndexSlot  bool
+	NeedsURI        bool
+	WeightedSupport bool
+}
+
+var CensusOrigins = map[models.CensusOrigin]CensusProperties{
+	models.CensusOrigin_OFF_CHAIN_TREE:          {Name: "offchain tree", NeedsDownload: true, NeedsURI: true},
```

Comment:
Suggest to change the `Name` hard-coded strings to constants

Reply:
Mmm, not sure... To me its clear to declare all properties (including human friendly name) on the same place.